templates/static_pages/help/accessibility-statement.html.tx

```diff
@@ -111,6 +90,24 @@
 
     <p>Many of our PDFs don’t meet accessibility standards - for example, they may not be structured so they’re accessible to a screen reader. This doesn’t meet WCAG 2.1 success criterion 4.1.2 (name, role value). We are currently fixing these PDFs.</p>
 
+    <h2 class="heading-medium">
+        Links
+    </h2>
+
+    <p>The purpose of a small number of links might not be clear to some user groups. This doesn’t meet WCAG 2.1 success criterion 2.4.4 Link Purpose (In Context). We plan to address this non-compliance shortly. 
+    
+    <h2 class="heading-medium">
+        Filters
+    </h2>
+
+    <p>A filter section may be confusing for screen reader users. This doesn’t meet WCAG 2.1 success criteria 1.3.1 Info and Relationships, or 2.4.6 Headings and Labels. We plan to address this non-compliance shortly. </p>
+
+    <h2 class="heading-medium">
+        Status messages
+    </h2>
+
+    <p>Messages which update the status of the page are not marked up programmatically for screen reader users. This doesn’t meet WCAG 2.1 success criterion 4.1.3 Status messages. We plan to address this non-compliance shortly.</p>
```

Comment:
Should this be `4.1.3 Status Messages` (capitalisation)?

Reply:
I copied it as it was written in the edited file that James added to the story and as Kayleigh also sent it to me.

Reply:
You might like to check this with Kayleigh then?

Reply:
It's been changed Vaughan